.github/ISSUE_TEMPLATE.md

```diff
@@ -12,6 +12,8 @@ Thank you!
 
 * **asyncpg version**:
 * **PostgreSQL version**:
+* **If you are using a PostgreSQL SaaS, then which, and can you reproduce
+  the issue with a local PostgreSQL install?**:
```

Comment:
Maybe reword this to: "Are you using a PostgreSQL SaaS?  If so, which? Can you reproduce the issue with a local postgres install?".